online-members/online-members.css

```diff
@@ -112,3 +112,14 @@
   display: flex;
   align-items: center;
 }
+
+footer {
+  display: block;
+  width: 100%;
```

Comment:
I don't think we need `width: 100%` since we already have display block

Reply:
I just copy-paste the CSS for now from other sites. As the main task was testing.